src/editor/attributes.js

```diff
@@ -30,4 +30,8 @@ export const attributes = {
         type: 'string',
         default: '',
     },
+    ccLangPref: {
+        type: 'array',
```

Comment:
Isn't this a single string value rather than an array?

Reply:
@AndreasMaros : That's correct. I was confused about the language object.